duckduckgo/macos-browser
# Rewrite BWEncryption in Swift

Task/Issue URL: https://app.asana.com/0/0/1204272370666896/f
CC: @tomasstrba 

**Description**:
Reimplement BWEncryption as Swift class and BWEncryptionOutput as Swift struct.

**Steps to test this PR**:
1. Run the app from develop and set up Bitwarden integration
1. Run the app from this branch and verify that Bitwarden integration works correctly
1. Remove `com.duckduckgo.macos.browser.debug.bitwarden.shared.key` entry from keychain and clean debug app data
1. Run the app from this branch, set up Bitwarden integration and verify that it works correctly
1. Verify that you can use Bitwarden passwords in the app.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
